appserver/connectors/connectors-internal-api/src/main/java/org/glassfish/connectors/config/validators/ConnectionPoolValidator.java

```diff
@@ -49,49 +50,58 @@
 
 /**
  * Implementation for Connection Pool validation.
- * Following validations are done :
- * - Validation of datasource/driver classnames when resource type is not null 
- * - Max pool size to be always higher than steady pool size
- * - Check if statement wrapping is on when certain features are enabled.
- * 
+ * Following validations are done:
+ * <ul>
+ * <li>Validation of datasource/driver classnames when resource type is not null
+ * <li>Max pool size to be always higher than steady pool size
+ * <li>Check if statement wrapping is on when certain features are enabled.
+ * </ul>
+ *
  * @author Shalini M
  */
-public class ConnectionPoolValidator
-    implements ConstraintValidator<ConnectionPoolConstraint, ResourcePool> {
-    
+public class ConnectionPoolValidator implements ConstraintValidator<ConnectionPoolConstraint, ResourcePool> {
```

Comment:
This class was updated too, because it uses same constants and same mechanism, despite it uses it's own implementation. I didn't create unit test for this.